documents/invoice/model.go

```diff
@@ -349,13 +328,20 @@ func (i *Invoice) Type() reflect.Type {
 	return reflect.TypeOf(i)
 }
 
-// CalculateDataRoot calculates the data root and sets the root to core document
-func (i *Invoice) CalculateDataRoot() ([]byte, error) {
+// DataRoot calculates the data root and sets the root to core document
+func (i *Invoice) DataRoot() ([]byte, error) {
```

Comment:
I would like to leave the method name here, calculate implies it modifies the document. It is not a simple getter. Don't you think that might be missed here?

Reply:
It doesn't calculate always. only when the data root is missing, else returns the already calculated one. Hence renamed